scanning/oscap-debug/sysctl-oval-eval-parallel.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/python3
+
+import time
+import subprocess
+import concurrent.futures
+
+from lib import util, results
+
+
+def run(fn):
+    handle = util.subprocess_Popen(['oscap', 'oval', 'eval', fn])
+    try:
+        rc = handle.wait(OSCAP_TIMEOUT)
```

Comment:
Maybe `util.subprocess_run(..., timeout=OSCAP_TIMEOUT)` would be more elegant?

Reply:
I need pid from process handle.